pkg/pillar/cas/cas.go

```diff
@@ -71,6 +71,9 @@ type CAS interface {
 	//GetImageHash: returns a blob hash of format <algo>:<hash> (currently supporting only sha256:<hash>) which the given 'reference' is pointing to.
 	// Returns error if the given 'reference' is not found.
 	GetImageHash(reference string) (string, error)
+	//s  returns the labels set on the image.
```

Comment:
Odd comment. What is it supposed to say?

Reply:
Sorry, typo. I will fix that

Reply:
When?